Triphouse Project/styles/main.css

```diff
@@ -0,0 +1,239 @@
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.nav_text a {
+  height: 100%;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+  color: #ffffff;
+}
+.nav_text a:hover {
+  border-bottom: 4px solid #f5bd41;
+}
+.nav_button {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 114px;
+  margin-left: 68px;
+}
+#nav_night {
+  width: 30px;
+  height: 30px;
+}
+.header_desc {
+  margin-top: 200px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 50px;
+  line-height: 59px;
+  color: #ffffff;
+}
+.input_sector {
+  margin-top: 100px;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 120px;
+}
+.header_input-labels{
+  margin-bottom: 24px;
+}
+label {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 21px;
+  color: #ffffff;
+  margin: 0px 240px 0px 10px;
+}
+.input_items{
+  display: flex;
+  flex-direction: row;
+  height: 65px;
+  background: #ffffff;
+  border-radius: 8px;
+}
+#input_citi{
+  padding-left: 25px;
+  width: 490px;
+  height: 100%;
+  border: none;
+  border-radius: 8px;
+  outline: none;
+}
+#input_citi:hover{
+  border: 4px solid #f5bd41;
+}
+#input_calendar{
+  padding-left: 25px;
+  width: 270px;
+  height: 100%;
+  border: none;
+  border-radius: 8px;
+  outline: none;
+}
+#input_calendar:hover{
+  border: 4px solid #f5bd41;
+}
+#input_amount{
+  padding-left: 25px;
+  width: 324px;
+  height: 100%;
+  border: none;
+  border-radius: 8px;
+  outline: none;
+}
+#input_amount:hover{
+  border: 4px solid #f5bd41;
+}
+.header_input-button{
+  width: 149px;
+  height: 100%;
+  border: none;
+  background: #3077C6;
+  border-radius: 8px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+  color: #FFFFFF;
+}
+.header_links{
+  margin: 130px auto 0px;
+}
+.header_links a{
+  margin: 0 8px;
+}
+/*------------------advantages------------------------*/
+.advantages {
+  width: 100%;
+  height: 570px;
+}
+.advantages_inner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.advantages_header {
+  margin-top: 120px;
+  margin-bottom: 80px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.advantages_header p {
+  margin-bottom: 20px;
+  font-size: 40px;
+  font-weight: 500;
+  color: #383838;
+}
+.advantages_header img {
+  width: 100px;
+  height: 4px;
+}
+.advantages_items {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+.advantages_item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.advantages_item img {
+  margin-bottom: 48px;
+}
+.advantages_item p {
+  font-size: 24px;
+  font-weight: 400;
+  color: #383838;
+}
+/*-------------------------------footer--------------------------------*/
+footer {
+  width: 100%;
+  height: 630px;
```

Comment:
Don't use fixed height, you can achive it fith content and gaps (padding,margin)